src/main/scala/ScalaModulePlugin.scala

```diff
@@ -140,6 +143,10 @@ object ScalaModulePlugin extends AutoPlugin {
         </developer>
       </developers>
     )
+  )
+
+  lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = Seq(
+    mimaPreviousVersion := None
   ) ++ mimaSettings ++ scalaModuleOsgiSettings
```

Comment:
Ah, good to know. I thought that MiMa probably doesn't support scala-js jars because it was already deactivated for Scala.js in the build, as the "previous version" is only set for the JVM project (https://github.com/scala/scala-parser-combinators/blob/1.0.x/build.sbt#L62-L64).

I guess the additional value is to also classfiles compiled from platform-specific sources?